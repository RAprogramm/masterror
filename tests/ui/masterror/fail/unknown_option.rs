use masterror::{AppCode, AppErrorKind, Masterror};

#[derive(Debug, Masterror)]
#[error("oops")]
#[masterror(code = AppCode::Internal, category = AppErrorKind::Internal, unknown)]
struct Unknown;

fn main() {}
