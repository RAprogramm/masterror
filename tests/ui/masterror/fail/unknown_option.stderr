error: unknown #[masterror] option `unknown`
 --> tests/ui/masterror/fail/unknown_option.rs:5:74
  |
5 | #[masterror(code = AppCode::Internal, category = AppErrorKind::Internal, unknown)]
  |                                                                          ^^^^^^^

warning: unused imports: `AppCode` and `AppErrorKind`
 --> tests/ui/masterror/fail/unknown_option.rs:1:17
  |
1 | use masterror::{AppCode, AppErrorKind, Masterror};
  |                 ^^^^^^^  ^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` (part of `#[warn(unused)]`) on by default
